proposals/all_proposals.md

```diff
@@ -0,0 +1,658 @@
+The wide usage of insecure communications (unencrypted emails and messenger apps (so-called OTTs), sms, others) have created the standard of communication for unaware citizens. In the very short-term (before end of 2018), the ePrivacy Regulation needs to be adopted with, among other clauses, one ensuring privacy by design and by default in all hardware and software used in the EU. 
+
+Oligopolies/Duopolies: 
+
+Nowadays (Google, Amazon, Facebook, Apple and Microsoft) GAFAM are by far the main actors that increasingly intermediate all aspects of our activity online. These companies are where internet and critical parts of the technology we use are developed. It has far-reaching implications: for example the existing duopoly of online advertisers (Facebook and Google) creates a huge pressure on online media and therefore affects how our societies are informed about public debate on crucial issues. In the short term competition law, hand in hand with strong data protection and privacy regulations, should be put in place immediately. In the longer term, alternatives to such tech giants should be encouraged and made available by the public sector.
+
+Right to science/Access to culture: 
+
+There is an urgent need to reduce maximalists approaches to copyright that favor widening of proprietary rights at the expense of noncommercial user activity. They jeopardize participation in culture and allow for introducing censorship with the excuse of fighting piracy, as the current discussions on the copyright Directive proposal in the EU show with the proposal of upload filters.
+
+2. What can be done to foster the privacy of European citizens and to protect their data from misuse by private corporations, states and third parties?
+
+Enhance data protection regulations worldwide (Convention 108 of the Council of Europe); ban data protection in trade agreements; ensure privacy regulations (in the EU: ePrivacy Regulation) ban limitless online and offline tracking, tracking walls, tracking cookies; work to improve our electronic communications to move away from current insecure communications by, for example, investing in ready-to-use encryption technologies for emails and make data protection and privacy by design and by default a requirement for all hardware and software sold in the EU.
+
+3. Which models of data ownership and governance can democratise the production and usage of data? When should data be open?
+
+Personal data needs to be protected with the highest protections we can achieve (Convention 108, GDPR….). Non-personal data should be open whenever possible so that everyone can re-use it. Specially, data produced by public institutions needs to be made open by default. We do not recommend models based on “data ownership” that envision the possibility to to trade personal data for services and benefits. It would affect the vunlnerable populations that have less resources and digital literacy and therefore enable forced trade of fundamental rights.
```

Comment:
possible tension between this view and later proposal from DSC Wiesbaden re: commercial payment of data providers? I see the proper valuation / remuneration of data being a big topic, and the concern raised by the authors here re: vulnerable populations is quite valid.